test/TESTING_GUIDELINES.md

```diff
@@ -55,6 +55,17 @@ Our UI tests also run on BrowserStack. If you would like to run them on your Bro
 - Specific to Safari on BrowserStack, you need to explicitly set the `localhostUrl` value in `config.json` to
   be read as `"localhostUrl": "https://bs-local.com:8080/",` this is due to the way Safari handles `https://localhost`.
 
+### Running tests with or without specific tags
+
+- We can now run tests that contain, or do not contain certain tags in the following way...
+
+`MOCHA_GREP="@1|@2" npm run test:ui` will run tests that ONLY have the tag @1 OR @2.
+`MOCHA_GREP="@3" npm run test:ui` will run tests that ONLY have the tag @3.
+`MOCHA_GREP='@1.*@5' npm run test:ui`will run tests that ONLY have the tags @1 AND @5.
+`MOCHA_INVERT="@3" npm run test:ui` will NOT run tests that have the tag @3.
+`MOCHA_INVERT="@1|@2" npm run test:ui` will NOT run tests that have the tags @1 OR @2.
+`MOCHA_INVERT='@1.*@3' npm run test:ui` will NOT run tests that have the tags @1 AND @3.
```

Comment:
Are we using these tags somewhere in the code or are these for future usage?

Reply:
Sorry, I'm OOO at the moment, but we will be using them in the future. I believe there was a PR merged recently for use on Percy, with the tag @percy